bears/general/CPDBear.py

```diff
@@ -30,15 +35,20 @@ class CPDBear(GlobalBear):
                      'Swift': 'swift'}
 
     LANGUAGES = set(language_dict.keys())
+    REQUIREMENTS = {
+        AnyOneOfRequirements(
+            [ExecutableRequirement('pmd'),
+             ExecutableRequirement('run.sh'),
+             ]
+        ),
+    }
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Duplication'}
 
     @classmethod
     def check_prerequisites(cls):
-        if which('bash') is None:
-            return 'bash is not installed.'
         if which('pmd') is None and which('run.sh') is None:
```

Comment:
while the error message isnt as good, the `AnyOneOfRequirements` above does the same thing.

And removing the `'bash'` prefix should be enough to make pmd work on Windows.